deployment/certs/generate-test-certs-keyvault.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+# This scripts generates test keys and certificates for the sample.
+# In a production environment such artifacts should be generated
+# by a proper certificate authority and handled in a secure manner.
+
+# Checking key vault parameter
+keyVault="$1"
+if [[ $keyVault == "" ]]; then
+    echo "Error: Missing key vault parameter"
+    exit 1
+elif ! az keyvault show -n $keyVault -o none; then
+    exit 1
+fi
+
+CERTS_DIR=./certs
+rm -rf $CERTS_DIR
+mkdir $CERTS_DIR
+TEMP_DIR=$CERTS_DIR/temp
+mkdir $TEMP_DIR
+
+generate_root_ca_cert() {
+    local certName=$1
+
+    ###########################
+    # Create a custom CA certificate to be used for 
+    # signing both cluster and client the certificate
+    ###########################
+
+    # Create custom CA-Certificate in Key-Vault
+    #CSR=$(az keyvault certificate create --vault-name $keyVault --name $certName --policy "@$certName-cert-policy.json" | jq -r '.csr')
```

Comment:
If we can generate the custom ca and sign the certs using az cli, that would be perfect right? Maybe this is something we can investigate in the future.

Reply:
Maybe we can create an open issue to keep track of this issue.

Reply:
Issue has been created. #4